components/index.tsx

```diff
@@ -0,0 +1,112 @@
+import styled from '@emotion/styled'
+import colors from '../colors'
+import { HeadingRenderer, ListRenderer } from './renderers'
+
+export const Root = styled.article({
+  padding: '2rem',
+})
+
+export const Paragraph = styled.p({
+  color: colors.greys[3],
```

Comment:
Curious if we can move `color` up to the `Root` component and let `p` tags just inherit the current color?